TradingOnTalentURAP/matching.py

```diff
@@ -0,0 +1,567 @@
+"UT":["University of Tampa", "University of Tennessee", "University of Texas at Austin", "University of Toledo"],
+"UTA":["University of Texas at Arlington"],
+"Utah":["University of Utah"],
+"UTB/TSC":["University of Texas at Brownsville and Texas Southmost College"],
+"UTC":["University of Tennessee at Chattanooga"],
+"UTD":["University of Texas at Dallas"],
+"UTEP":["University of Texas at El Paso"],
+"UTM":["University of Tennessee at Martin"],
+"UTPB":["University of Texas of the Permian Basin"],
+"UTRGV":["University of Texas Rio Grande Valley"],
+"UTSA":["University of Texas at San Antonio"],
+"UVA":["University of Virginia"],
+"UVM":["University of Vermont"],
+"UVU":["Utah Valley University"],
+"UW":["University of Washington", "University of Wisconsin system", "University of Wisconsin–Madison", "University of Wyoming"],
+"UWF":["University of West Florida"],
+"UWG":["University of West Georgia"],
+"UWGB":["University of Wisconsin–Green Bay"],
+"UWM":["University of Wisconsin–Milwaukee"],
+"Valpo":["Valparaiso University"],
+"Vandy":["Vanderbilt University"],
+"VCU":["Virginia Commonwealth University"],
+"Virginia Tech":["Virginia Polytechnic Institute and State University"],
+"VMI":["Virginia Military Institute"],
+"VPI":["Virginia Polytechnic Institute and State University"],
+"VSU":["Valdosta State University", "Virginia State University"],
+"VT":["Virginia Polytechnic Institute and State University"],
+"VPI":["Virginia Polytechnic Institute and State University"],
+"VTC":["Vermont Technical College"],
+"VU":["Valparaiso University", "Vanderbilt University", "Villanova University", "Vincennes University", "Vanguard University"],
+"W&J":["Washington & Jefferson College"],
+"W&L":["Washington and Lee University"],
+"W&M":["The College of William & Mary"],
+"Wash U":["Washington University in St. Louis"],
+"WUSTL":["Washington University in St. Louis"],
+"Wazzu":["Washington State University"],
+"WC":["Wagner College", "Wilmington College"],
+"WCU":["Western Carolina University", "West Chester University"],
+"West Point":["U.S. Military Academy"],
+"Western":["Western Washington University"],
+"WFU":["Wake Forest University"],
+"Williams":["Williams College"],
+"WIU":["Western Illinois University"],
+"WKU":["Western Kentucky University"],
+"WMU":["Western Michigan University"],
+"WPI":["Worcester Polytechnic Institute"],
+"WSU":["Washington State University", "Wichita State University", "Wright State University"],
+"WSSU":["Winston-Salem State University"],
+"WTAMU":["West Texas A&M University"],
+"WVU":["West Virginia University"],
+"WVUP":["West Virginia University at Parkersburg"],
+"WVU Parkersburg":["West Virginia University at Parkersburg"],
+"WWU":["Western Washington University"],
+"XU":["Xavier University (Cincinnati)"],
+"XULA":["Xavier University of Louisiana"],
+"The Y":["Brigham Young University"],
+"YSU":["Youngstown State University"],
+"YU":["Yeshiva University"]}
+
+def processor(oag,schools,schools_d,colloquial_names):
```

Comment:
'oag' stands for 'open academic graph, so you should choose a more descriptive/relevant name for this task.